src/theme/Footer/index.js

```diff
@@ -82,8 +82,16 @@ function Footer() {
         dangerouslySetInnerHTML={{
           __html: copyright
         }} /> : null}
+        <div className="footer__legal" >
+          <ul>
+            <li><a href="https://unity.com/legal">Legal</a></li>
+            <li><a href="https://unity.com/legal/privacy-policy">Privacy Policy </a></li>
+            <li><a href="https://unity.com/legal/cookie-policy">Cookies</a></li>
+            <li><a href="https://unity.com/legal/do-not-sell-my-personal-information">Do Not Sell or Share My Personal Information</a></li>
+         </ul>
```

Comment:
Missing the OneTrust cookie settings link:

```html
<!-- OneTrust Cookies Settings button start -->
<li><a href="javascript:void(0);" id="ot-sdk-btn" class="ot-sdk-show-settings">Your Privacy Choices (Cookie Settings)</a></li>
<!-- OneTrust Cookies Settings button end -->
```

Reply:
Thank you!